spec.idl

```diff
@@ -470,6 +471,10 @@ interface CatchClause : Node {
   attribute Block body;
 };
 
+interface ConstDeclaration : Delcaration {
+  attribute VariableDeclaration declaration;
```

Comment:
This should be a list of declarators. It should be a VariableDeclaration analogue.